docs/vrf/v2/best-practices.md

```diff
@@ -96,3 +96,65 @@ function fulfillRandomWords(
   s_requestIndexToRandomWords[requestNumber] = randomWords;
 }
 ```
+
+## Processing VRF responses through different execution paths
+
+If you want to process VRF responses depending on predetermined conditions, you can create an `enum`. When requesting for randomness, map each `requestId` to an enum. This way, you can handle different execution paths in `fulfillRandomWords`.
+
+```solidity
+enum Logic { A, B, C}
+
+uint256 public variableA;
+uint256 public variableB;
+uint256 public variableC;
+
+mapping(uint256 => Logic) public requests;
+
+function updateVariableA() public {
```

Comment:
Could we create a single `updateVariable` function that takes in the variable to update?